segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkStorageMetrics.java

```diff
@@ -94,4 +94,8 @@
     static final Counter SLTS_SYSTEM_TRUNCATE_COUNT = STATS_LOGGER.createCounter(MetricsNames.SLTS_SYSTEM_TRUNCATE_COUNT);
 
     static final Counter LARGE_CONCAT_COUNT = STATS_LOGGER.createCounter(MetricsNames.STORAGE_LARGE_CONCAT_COUNT);
+
+    static final OpStatsLogger SLTS_HEALTH_PARALLEL_THROTTLE = STATS_LOGGER.createStats(MetricsNames.SLTS_HEALTH_PARALLEL_THROTTLE);
```

Comment:
i remember from some other exercise that we have to call close on these metrics when for example a container's close gets invoked to avoid stale cached values from being read, but could be a different exercise altogether.